### PR TITLE
Made button sit in line on all screen sizes

### PR DESCRIPTION
**Pull request by TurnbulA**
Used relative positioning
required me moving the button div into the header div 
Had to go into responsive section and make some changes 
Button now sits inline with the bottom of the header no matter the screen size

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -520,15 +545,12 @@ option {
...
-  .section-form{
-    height: 150rem;
-  }
   form{
     width: 100%;
     margin-bottom: 5rem;
   }
   input{
-    font-size: 4rem;
+    font-size: 1.6rem;
```
Comment (mali891) [inline style.css:552]: Is this required to work position the header button?

File: style.css [hunk 2]
```diff
@@ -520,15 +545,12 @@ option {
     padding: 0;
     width: 100%;
   }
-  .section-form{
```
Comment (mali891) [inline style.css:523]: Is this required to work position the header button?

File: style.css [hunk 3]
```diff
@@ -492,7 +516,8 @@ option {
   .section-form{
     flex-direction: column;
     justify-content: center;
-    height: 40rem;
+    height: auto;
```
Comment (mali891) [inline style.css:518]: Is this required to work position the header button?

File: style.css (hunk 1 above)
Comment (TurnbulA) [reply style.css:552]: It was some work I did before the ticket came through

Comment (mali891) [reply style.css:552]: @TurnbulA As discussed, please only commit one feature per PR.

**Review by mali891 — APPROVED** [review]

**Pull request merged**